DesignSafe-CI/portal
# task/DES-2939 -  New App Notes Field "hideQueue"

## Overview: ##
Added a new field for Apps that will hide the queue selection for apps. This is intended only for use with Compress and Extract apps

## PR Status: ##


## Related Jira tickets: ##

* [DES-2939](https://tacc-main.atlassian.net/browse/DES-2939)

## Summary of Changes: ##
Added a boolean that hides the app's queue and allocation if set to `true`.

## Testing Steps: ##
1. Go to designsafe.dev/rw/workspace/compress
2. Check to make sure that the Queue and Allocation selection menus are **not** available through the job submission pipeline
3. Go to any other app (other than Extract) and make sure that the Queue and Allocation selection menus **are** available

## UI Photos:

## Notes: ##
